libexec/bats-exec-test

```diff
@@ -16,15 +16,7 @@ if [ "$1" = "-x" ]; then
 fi
 
 BATS_TEST_FILENAME="$1"
-if [ -z "$BATS_TEST_FILENAME" ]; then
```

Comment:
I'm in favor of keeping this as a defense mechanism against bats-exec-test getting called incorrectly.

Reply:
Do you mean `bats-exec-test` could be called directly?